setup.py

```diff
@@ -52,7 +52,7 @@ def get_version(package):
     packages=find_packages(exclude=['tests*']),
     include_package_data=True,
     install_requires=[],
-    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*,",
```

Comment:
This will need rebasing. (Hence the "after" 🙂)

Reply:
yes I will do that, and also drop 3.4 from dj11 as well